cap/modules/schemas/cli.py

```diff
@@ -34,59 +34,69 @@
 
 from cap.modules.fixtures.cli import fixtures
 
-from .models import Schema
+from .models.schemas import Schema
+from .models.templates import Template
 
 
 @fixtures.command()
 @with_appcontext
-@click.option('--dir',
-              '-d',
+@click.option('--dir', '-d',
               type=click.Path(exists=True),
               default='cap/modules/fixtures/schemas')
 def schemas(dir):
     """Load default schemas."""
+    load_from_files(dir, fixture='schema')
+
+
+@fixtures.command()
+@with_appcontext
+@click.option('--dir', '-d',
+              type=click.Path(exists=True),
+              default='cap/modules/fixtures/templates')
+def templates(dir):
+    """Load default templates."""
+    load_from_files(dir, fixture='template')
```

Comment:
this should be one method, as explained before, templates have to be always installed before schemas (otherwise doesnt work)